Night_Mode.py

```diff
@@ -644,6 +644,8 @@ def nm_switch():
             nm_off()
         else:
             nm_on()
+        nm_save()
```

Comment:
Why trigger save in there?

Reply:
So I could check for setting changes with mw.pm.profile.get() in one go.
I guess you would prefer: runHook("night_mode_state_changed", nm_state_on) ?

Reply:
Yes, that might be better